lib/streams/json-parse-stream.js

```diff
@@ -0,0 +1,24 @@
+var Parser = require('jsonparse');
+var ZTransform = require('../transform');
+var util = require('util');
+
+function JSONParseStream (options) {
+  var self = this;
+  this.parser = new Parser();
+
+  if(!options) options = {};
+  if(!options.objectMode) options.objectMode = true;
```

Comment:
This is wrong.  A JSON parse stream should be writableObjectMode true but readableObjectMode false.  Setting objectMode to true will set both readable and writable object modes to true.